pkg/pd/etcd.go

```diff
@@ -45,7 +45,7 @@ func init() {
 	_ = zap.RegisterEncoder("etcd-client", newZapEncoder)
 }
 
-func NewEtcdClient(lc fx.Lifecycle, config *config.Config) (*clientv3.Client, error) {
+func NewEtcdClientNoLC(config *config.Config) (*clientv3.Client, error) {
```

Comment:
Maybe you can create an fx app and then introduce this as dependency, so that no need to introduce extra function.

Reply:
Maybe the  complexity is increased. So far `kvauth` is very simple.